src/plugins/advanced_settings/public/plugin.ts

```diff
@@ -116,6 +126,44 @@ export class AdvancedSettingsPlugin
       },
     ]);
 
+    if (core.chrome.navGroup.getNavGroupEnabled()) {
+      setupUserSettingsPage(contentManagementSetup);
+
+      core.application.registerAppUpdater(this.appUpdater$);
```

Comment:
```suggestion
```
No need to registerAppUpdater for all of the apps because you have already add the updater$ when register the application.